solutions/devsprint-bruno-faganello-2/SpaceApp/Screens/NextLaunchView.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  NextLaunchView.swift
+//  SpaceApp
+//
+//  Created by Paolo Prodossimo Lopes on 26/04/22.
+//
+
+import SwiftUI
+
+struct NextLaunchView: View {
```

Comment:
[REQUEST CHANGED]
This View Should have a List to show All Next Launch
I think you need to create a Cell NextLaunchRow to help you to create this View

Reply:
Named changed for NextLaunchViewCell because is only component to present on HomeView